### PR TITLE
[NO-TICKET] Adding dependabot to mgov for testing

### PR DESCRIPTION
**Pull request by line47**
This dependabot should check for NPM updates for any of the design system packages.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  # new versions are published to the npm registry - checks daily
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'daily'
+    default_labels:
+      - "Dependabot"
+      - "Design System Update"
+    allow:
+      - dependency-name: "@cmsgov/design-system"
```
Comment (line47) [inline .github/dependabot.yml:13]: Checking for all design system updates

**Review by scheul93 — APPROVED** [review]
Nice, thanks for adding this!
Since the check is done daily, not sure if it'll catch today's core 2.7.0 release (depending on what time it does the daily check)

**Comment by line47** [comment]
Hoping it's set up correctly 🤞

**Comment by line47** [comment]
Looks like there was an issue with the dependabot set up @scheul93 
 https://github.com/CMSgov/mgov-design-system/runs/2943267458

**Pull request merged**